src/dsverifier.cpp

```diff
@@ -1074,7 +1096,7 @@ std::string prepare_bmc_command_line_ss()
     command_line =
         model_checker_path
             + "/esbmc input.c --no-bounds-check --no-pointer-check "
-              "--no-div-by-zero-check -DBMC=ESBMC -I "
+              "--no-div-by-zero-check --smt-during-symex  --smt-symex-guard --z3 -DBMC=ESBMC -I "
```

Comment:
Can you please add some comments to justify the use of these parameters?

Reply:
I will change that. However, we will not use that in this paper.

Reply:
I believe you can use --solver in order to invoke z3, boolector, etc instead of using "z3" directly on the command line.

--solver z3
--solver boolector